pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>com.github.ONSdigital</groupId>
             <artifactId>restolino</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>6cf0bd10f3b30a2bde866beaf0a75c7f2f74f6a1</version>
```

Comment:
We should probably create a release version of this.

Reply:
That is my intention. The updated version of restolino is going to have a release version. I can tag this commit if you think it's better, but my intention was to ensure the the exact commit is the same as currently without needing to change anything else.